skeleton_scary_game.py

```diff
@@ -0,0 +1,151 @@
+import time
+import random
+
+# This is where the actual game will run from
+class game_engine(object):
```

Comment:
class names should be capitalized see this: https://stackoverflow.com/questions/14973963/if-the-convention-in-python-is-to-capitalize-classes-why-then-is-list-not-cap

Also from this PR and forward you're going to use the Google Python style guide for your code:
https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md